charts/hivemq-swarm/values.yaml

```
@@ -0,0 +1,240 @@
+  ##
+  env: [ ]
+
+  ## Additional volumes to the Swarm pod.
+  ## By default: ephemeral volume for scenarios + previous run persistence
+  ##
+  extraVolumes:
+    - emptyDir: { }
+      name: scenario-persistence
+
+  # Adding a HiveMQ swarm license (requires also the VolumeMount for the HiveMQ Swarm pod)
+  #  - name: license
+  #    secret:
+  #      secretName: swarm-license
+
+  ## Extra volume mounts that will be added to the Swarm container
+  ##
+  extraVolumeMounts:
+    - mountPath: /scenario
+      name: scenario-persistence
+
+  # Mounting a HiveMQ swarm license (requires also the extraVolume for the HiveMQ Swarm pod)
+  #    - name: license
+  #      readOnly: true
+  #      mountPath: /license
+
+  ## Additional persistent volumes to create/mount for the commander StatefulSet
+  ##
+  volumeClaimTemplates: [ ]
+
+## HiveMQ Swarm Agents configuration
+##
+agents:
+  logLevel: info
+  agentCount: 3
+
+  ## Configure resources for HiveMQ Swarm Agent Pods (Note: request = limits for HiveMQ Swarm)
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    cpu: 500m
+    memory: 1G
+
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ##
+  podSecurityContext:
+    sysctls:
+      - name: net.ipv4.ip_local_port_range
+        value: "1024 65535"
+
+  ## Configure HiveMQ Swarm Agent Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  securityContext: {}
+
+  ## Configure affinity preset for HiveMQ Swarm Agent Pods
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node
+  ##
+  affinity: {}
+    # podAntiAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     # Try not to co-locate with other swarm instances
+    #     - weight: 50
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: swarm-task
+    #               operator: In
+    #               values:
+    #                 - ${LABEL}
+    #         topologyKey: "kubernetes.io/hostname"
+    #     # Try not to co-locate with HiveMQ nodes
+    #     - weight: 60
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchExpressions:
+    #             - key: app
+    #               operator: In
+    #               values:
+    #                 - hivemq
+    #         topologyKey: "kubernetes.io/hostname"
+  # Add environment variables for all agents
+  env: [ ]
+
+  ## Additional volumes for the HiveMQ Swarm agent pod.
+  extraVolumes: [ ]
+
+  ## Extra volume mounts that will be added to the swarm agent container
+  extraVolumeMounts: [ ]
+
+## Monitoring configuration
+##
+monitoring:
+  # Deploy ServiceMonitor for Prometheus Operator and enable Prometheus
```

```suggestion
  # Deploy ServiceMonitor for Prometheus Operator and  Configmap for Grafana Dashboard
```

The flag will deploy both ServiceMonitor and the Grafana Dashboard - maybe it's worth mentioning as otherwise it's not clear where the dashboard appears from